src/k8s/pkg/k8sd/features/cilium/chart.go

```diff
@@ -46,4 +46,8 @@ var (
 
 	// ciliumOperatorImageTag is the tag to use for the cilium-operator image.
 	ciliumOperatorImageTag = "1.17.1-ck0"
+
+	ciliumDefaultVXLANPort int = 8472
```

Comment:
```suggestion
	ciliumDefaultVXLANPort = 8472
```

Do we need to put `int` here explicitly?